qiskit_nature/operators/second_quantization/spin_op.py

```diff
@@ -96,12 +96,13 @@ class SpinOp(SecondQuantizedOp):
 
         "X_0"
         "Y_0^2"
-        "Y_1^2 Z_1^3 X_0^1 Y_0^2 Z_0^2"
+        "Y_0^2 Z_0^3 X_1^1 Y_1^2 Z_1^2"
 
     are possible labels.
     For each :code:`index` the operations `X`, `Y` and `Z` can only be specified exclusively in
     this order. `+` and `-` are same order with `X` and cannot be used with `X` and `Y`.
```

Comment:
```suggestion
    this order. `+` and `-` are same order with `X` and cannot be used with `X` and `Y`
    because ladder operators will be parsed into `X` and `Y`.
```